rest/src/main/kotlin/json/request/ChannelRequests.kt

```diff
@@ -53,6 +53,15 @@ public data class ChannelModifyPatchRequest(
     val videoQualityMode: Optional<VideoQualityMode?> = Optional.Missing(),
     @SerialName("default_auto_archive_duration")
     val defaultAutoArchiveDuration: Optional<ArchiveDuration?> = Optional.Missing(),
+    @SerialName("default_reaction_emoji")
+    val defaultReactionEmoji: Optional<DiscordDefaultReaction> = Optional.Missing(),
```

Comment:
```suggestion
    val defaultReactionEmoji: Optional<DiscordDefaultReaction?> = Optional.Missing(),
```
docs say it's also nullable